### PR TITLE
[PRM-1109]: messages clean up & 'You can appeal this penalty' page messages

### PR DESCRIPTION
**Pull request by BackRowOla**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -13,8 +13,6 @@ common.radioOption.no = No
 
 app.common.back = Back
 
-error.summary.title = There is a problem
```
Comment (LewisMitchell) [inline conf/messages:16]: This is being used in summary.scala.html and errorSummary.scala.html

File: conf/messages [hunk 2]
```diff
@@ -115,13 +113,6 @@ honestyDeclaration.lossOfStaff = of loss of staff essential to the VAT process
 agent.honestyDeclaration.lossOfStaff = my client was affected by loss of staff essential to the VAT process
 honestyDeclaration.other =
 
-# Errors
-# ----------------------------------------------------------
-error.prefix = Error:
```
Comment (LewisMitchell) [inline conf/messages:120]: This is being used in fileUploadDialog.scala.html

File: conf/messages [hunk 3]
```diff
@@ -115,13 +113,6 @@ honestyDeclaration.lossOfStaff = of loss of staff essential to the VAT process
...
-# Not sure if this is as per design
-error.clickAcceptAndContinue = Please click ’Accept and continue’
-error.clickContinue = Please click ’Continue’
```
Comment (LewisMitchell) [inline conf/messages:123]: Arguably you can get rid of these two, but you'd also have to remove the form validation behind it (the form validation is quite redundant tbh)

File: conf/messages [hunk 4]
```diff
@@ -562,10 +553,23 @@ duplicateAppeal.p1 = You cannot submit a duplicate request for the same penalty.
...
+# Can you appeal Page
+# ----------------------------------------------------------
+youCanAppealThisPenalty.title = You can appeal this penalty online
+youCanAppealThisPenalty.heading = You can appeal this penalty online
+youCanAppealThisPenalty.p1 = If HMRC reviews this penalty and accepts your appeal, we will remove:
+youCanAppealThisPenalty.li.1 = this penalty
+youCanAppealThisPenalty.li.2 = all other VAT late submission or late payment penalties for the same period
+youCanAppealThisPenalty.p2 = Do you want to appeal this penalty?
+youCanAppealThisPenalty.error.required = Select yes if you want to appeal this penalty
+
```
Comment (LewisMitchell) [inline conf/messages:565]: I'm adding this under PRM-1117

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
I'm just thinking about this and if we decide to keep the things I raised, then we could close the PR?

File: conf/messages (hunk 1 above)
Comment (BackRowOla) [reply conf/messages:16]: Was a duplicate text. Its still in the document on line 569

File: conf/messages (hunk 2 above)
Comment (BackRowOla) [reply conf/messages:120]: Was a duplicate text. Its still in the document on line 568

Comment (LewisMitchell) [reply conf/messages:120]: 😮

File: conf/messages (hunk 1 above)
Comment (LewisMitchell) [reply conf/messages:16]: 😮

File: conf/messages (hunk 3 above)
Comment (LewisMitchell) [reply conf/messages:123]: ignore

File: conf/messages (hunk 4 above)
Comment (BackRowOla) [reply conf/messages:565]: I'll remove this. Forgot to do so after creating the excel sheet.

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**